build-copy-source.js

```diff
@@ -0,0 +1,28 @@
+// A simple script to copy over source files as *.js.flow
+// so that flow will be happy with the distributed package.
+
+const fs = require('fs');
+const path = require('path');
+
+// little recursive walk
+const walk = (dir, fn) => {
+    fs.readdirSync(dir).forEach((name) => {
+        const full = path.join(dir, name);
+        if (fs.statSync(full).isDirectory()) {
+            walk(full, fn);
+        } else {
+            fn(full);
+        }
+    });
+};
+
+const src = __dirname + '/src';
+const dist = __dirname + '/dist';
```

Comment:
Captain nitpick says this won't run on Windows... but that's probably irrelevant.